plugin/output/splunk/splunk.go

```diff
@@ -98,6 +100,14 @@ func (p *Plugin) Start(config pipeline.AnyConfig, params *pipeline.OutputPluginP
 	p.config = config.(*Config)
 	p.client = p.newClient(p.config.RequestTimeout_)
 
+	ctx := context.TODO()
```

Comment:
Background ?

Reply:
It should pass from pipeline in future, therefore TODO